Automattic/jetpack
# VideoPress: Show site default privacy setting in video block control

Fixes #27737

## Proposed changes:

* Exposes a `jetpackVideoPressSiteSettings` object to the block editor in the same way that `jetpackVideoPressInitialState` is exposed to the admin dashboard, using it to determine the correct label for the default site settings at load time.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the VideoPress dashboard, set the default privacy level to `Public` and wait some seconds for the change to be persisted
* Go to the block editor, add a `VideoPress` block and upload/select a video
* In the `Privacy and rating` section of the block control panel, check that the site default option is presented as `Site Default (Public)` 
* Go to the VideoPress dashboard, set the default privacy level to `Private` and wait some seconds for the change to be persisted
* Go back to the block editor and hard refresh the page
* * In the `Privacy and rating` section of the block control panel, check that the site default option is presented as `Site Default (Private)` 

Before | After
-|-
![2023-01-24_14-45-33](https://user-images.githubusercontent.com/8486249/214368699-e5654517-dfbf-445f-bb70-3249df8bf5b4.png) | ![2023-01-24_14-40-59](https://user-images.githubusercontent.com/8486249/214368494-140d7241-8f84-4e87-9b66-9e0eed489b2c.png)
ㅤ| ![2023-01-24_14-39-44](https://user-images.githubusercontent.com/8486249/214368502-294a54ef-3dc5-4d31-a971-cc32e8133a8b.png)
